jest.config.ts

```diff
@@ -1,4 +1,4 @@
-const transformNodeModules = ['react-native', '@react-native']
+const transformNodeModules = ['react-native', '@react-native', 'react-navigation-header-buttons']
```

Comment:
because the jests fail when you import this library because a module (of this lib) can't be exported if i remember right. i can provide some error message if you want.

Reply:
not it's fine, thanks.